Dockerfile.template

```diff
@@ -8,7 +8,6 @@ RUN apt-get update \
       && apt-get install -y --no-install-recommends \
            postgresql-$PG_MAJOR-postgis-$POSTGIS_MAJOR=$POSTGIS_VERSION \
            postgresql-$PG_MAJOR-postgis-$POSTGIS_MAJOR-scripts=$POSTGIS_VERSION \
-           postgis=$POSTGIS_VERSION \
```

Comment:
Can we discuss this removal? I know they don't version this package name, so is the issue that only the 2.4 version is now available? If so, then I'd like to remove 2.3 entirely.

Reply:
I just saw that you commented on this already.

The `postgis` package provides a number of PostGIS binaries if I recall. There is some discussion in https://github.com/appropriate/docker-postgis/issues/8 and https://github.com/appropriate/docker-postgis/issues/26